modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/submissions_controller.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module V0
+    class SubmissionsController < ApplicationController
```

Comment:
Let's solidify the name of this resource that our API exposes as `/claim_submissions`.